knowledge-base/error-in-web-site-project-failed-to-setup-service-system-invalidcastexception.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Error is thrown on attempt to add Report Viewer control on to the page.
```

Comment:
If we know the exact error thrown, we must add it in the meta tags here. For example, I would change this to: "Failed to setup service when adding Report Viewer control on the page."